main.go

```diff
@@ -99,24 +121,45 @@ func main() {
 				secondAmount, _ := secondResultsCosts[service]
 				secondAmount *= multiplier
 				delta := secondAmount - amount
-				finalResultsCosts[service] = []float64{
+				finalResultsCosts = append(finalResultsCosts, ServiceCosts{
+					service,
 					amount,
 					secondAmount,
 					delta,
-				}
+				})
 				totalAmount += amount
 				totalSecondAmount += secondAmount
 				totalDelta += delta
 			}
+
+			// Sort results
+			sort.Slice(finalResultsCosts, func(i, j int) bool {
+				var retVal bool
+				switch sortColumn {
+				case "start":
+					retVal = finalResultsCosts[i].amount < finalResultsCosts[j].amount
+				case "end":
+					retVal = finalResultsCosts[i].secondAmount < finalResultsCosts[j].secondAmount
+				case "delta":
+					retVal = finalResultsCosts[i].delta < finalResultsCosts[j].delta
```

Comment:
For delta, I assume this would sort like so:

$500, $250, $5, -$10, -$100, -$1000

I wonder if it might make sense to ignore the +/- on delta - since you're probably interested in the biggest swings in cost change (in either direction).

Eg. -$1000, $500, $250, -$100, -$10, $5

Reply:
Hmm, interesting. I would think primary use cases would be looking for biggest cost increases or biggest savings.

But maybe there could be another sort mode for this, to see biggest swings. Not sure what to call it, something like ascending-absolute-value?

Reply:
@stevennoto Sure, I think something like that would work for me!

For context my use case is to semi-automate more of the data gathering for my monthly AWS spending reports. I care more about big swings (in either direction) than top increases or top decreases separately. I use those "top changes" to guide my investigation and documenting of why those changes happened.